data/mozo.ui

```diff
@@ -37,701 +197,416 @@
       </popup>
     </ui>
   </object>
-  <object class="GtkMenu" constructor="uimanager1" id="edit_menu">
+  <object class="GtkImage" id="undo-icon">
     <property name="visible">True</property>
-    <property name="can_focus">False</property>
+    <property name="can-focus">False</property>
+    <property name="icon-name">undo</property>
```

Comment:
we use edit-undo
https://developer.gnome.org/gtk3/stable/gtk3-Stock-Items.html#GTK-STOCK-UNDO:CAPS